### PR TITLE
CONTRAST-38383 - Added Pylons documentation

### PR DESCRIPTION
**Pull request by jleo32**

### CODE REVIEW

File: content/installation/python/PythonInstall.md [hunk 1]
```diff
@@ -101,6 +101,16 @@ app = get_wsgi_application()
 app = ContrastMiddleware(app)
 ```
 
+#### Pylons
+
+``` python
+from pylons.wsgiapp import PylonsApp
+from contrast.agent.middlewares import WSGIMiddleware
```
Comment (peter-lazorchak) [inline content/installation/python/PythonInstall.md]: This should be `from contrast.agent.middlewares.wsgi_middleware import WSGIMiddleware`. For consistency I also think we should add `as ContrastMiddleware` (a few lines down too)

Comment (jleo32) [reply content/installation/python/PythonInstall.md]: Let me update that!

Comment (jleo32) [reply content/installation/python/PythonInstall.md]: udpated

Comment (jleo32) [reply content/installation/python/PythonInstall.md]: spelling is hard

**Pull request merged**